plugin.json

```diff
@@ -58,6 +58,13 @@
                     }
                 ]
             },
+            {
+                "key": "JitsiCompatibilityMode",
+                "display_name": "Enable compatibility mode",
+                "type": "bool",
+                "help_text": "(Insecure) This improve the compatibility with your jitsi instance using jitsi javascript code through a proxy build in the plugin. **WARNING:** This can compromise the security of your mattermost instance if the security of your Jitsi instance get compromised.",
+                "default": false
```

Comment:
New setting off by default

Reply:
Is there a use case where an admin wants to turn this on?

Reply:
If you have a version out of sync with our JavaScript copy, you can find compatibility issues, in that case you can decide to enable it until is on sync again

Reply:
I have many questions, might be easier to chat over video next week. At least:
1. Why/when would that happen?
2. How likely would this happen?
2. Is it easy to resolve?
3. Is it unique to the Jitsi plugin, or does it happen with others as well?

Reasons why I have the above questions is to better understand if the setting is needed (e.g. instead of an insecure option to the admin, should the plugin fail and log an error message explaining why), and what the text should say.